{{cookiecutter.project_shortname}}/code/.gitkeep

```diff
@@ -0,0 +1 @@
+# This file exists only to add templates/ folder to git repository.
```

Comment:
Minor: naming-wise, `code` gives me the Puppet shivers, but I understand that it's human/non-developer-friendly. An alternative could also be `modules` (given that this might contain more packages/folders).

Reply:
maybe /src?